requirements/pip.txt

```diff
@@ -42,7 +42,7 @@ celery==4.1.1  # pyup: ignore
 
 django-allauth==0.40.0
 
-GitPython==3.0.4
+GitPython==3.0.8
```

Comment:
Do we still need this when we're running all the commands in Docker now?

Reply:
We still need it, we use it to parse tags and branches (this happens outside the container)